roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -59,28 +59,29 @@ enum ApprovalCheckResult {
 }
 
 enum ApprovalVotingMessage {
-    /// Check if the assignment is valid and can be accepted by our view of the protocol.
-    /// Should not be sent unless the block hash is known.
-    CheckAndImportAssignment(
-        IndirectAssignmentCert,
-        ResponseChannel<AssignmentCheckResult>,
-    ),
-    /// Check if the approval vote is valid and can be accepted by our view of the
-    /// protocol.
-    ///
-    /// Should not be sent unless the block hash within the indirect vote is known.
-    CheckAndImportApproval(
-        IndirectSignedApprovalVote,
-        ResponseChannel<ApprovalCheckResult>,
-    ),
-    /// Returns the highest possible ancestor hash of the provided block hash which is
-    /// acceptable to vote on finality for.
-    /// The `BlockNumber` provided is the number of the block's ancestor which is the
-    /// earliest possible vote.
-    ///
-    /// It can also return the same block hash, if that is acceptable to vote upon.
-    /// Return `None` if the input hash is unrecognized.
-    ApprovedAncestor(Hash, BlockNumber, ResponseChannel<Option<Hash>>),
+	/// Check if the assignment is valid and can be accepted by our view of the protocol.
```

Comment:
we use inconsistent indentation for rust code in the guide
mostly spaces, but here it's tabs